man/foreign.doc

```diff
@@ -2187,6 +2211,43 @@ As PL_get_list(), but raises a type or instantiation error if
 As PL_get_nil(), but raises a type or instantiation error if
 \arg{t} is not the empty list.
 
+    \cfunction{int}{PL_cvt_i_bool}{term_t p, int *c}
```

Comment:
Good to add these.    I think they should go to a new section, probably together with the _o_ versions and some explanation about the rationale behind these.   I can take care of that.

Reply:
OK, I'll leave this as-is, on the assumption that you can easily cherry-pick the PR. You might want to review the comment at line 2128. I've also added a comment about "generic" here:
https://github.com/SWI-Prolog/packages-cpp/pull/24/files
SWI-cpp2.h line 341 (the definition of PlTerm::integer(...))
The test cases are in https://github.com/SWI-Prolog/packages-cpp/pull/24 and I'll modify them as well.